deployment/terraform/outputs.tf

```
@@ -5,3 +5,7 @@ output "edc_host" {
 output "assets_storage_account" {
   value = azurerm_storage_account.assets.name
 }
+
+output "did_endpoint" {
+  value = "${azurerm_storage_account.did.primary_web_endpoint}${azurerm_storage_blob.did.name}"
```

Just double checking that a "/" is not required:
```suggestion
  value = "${azurerm_storage_account.did.primary_web_endpoint}/${azurerm_storage_blob.did.name}"
```

---

I tested it before. / is already included in the primary_web_endpoint but I can add it for clarity.
The fact that the did_endpoint is correct is also implicitly tested in the workflow when calling the endpoint.